src/ArmTemplates/Extractor/Models/ExtractorParameters.cs

```diff
@@ -1,56 +1,133 @@
-﻿namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Models
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.FileHandlers;
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Models
 {
-    public class ExtractorParameters
+    /// <summary>
```

Comment:
Nit: I'm not sure this summary adds any value. I'd just remove it.

Reply:
removed